### ISSUE TITLE
Values not read propperly

### ISSUE THREAD

**Issue by Dandrum** [FynniX/acc-node-wrapper#4]
Hi,

semes Like any value after 

https://github.com/FynniX/acc-node-wrapper/blob/6fa0bde696d5f0a190d9a4bbbece821ab8213902/src/index.js#L709

Are not working even if i log out the direct reader value...
I like to have this values but they not working: 
```js
result.ignitionOn = reader.ReadUInt32() > 0
result.starterEngineOn = reader.ReadUInt32() > 0
result.isEngineRunning = reader.ReadUInt32() > 0
```

**Comment by QuadratoTV** [comment]
This issue is caused because the buffer doesnt read the full length of the mmapped file as the variable "m_physics_length" was set up for an earlier acc version. I encountered this today as well and fixed it by changing it from the default value of 712 to 790.

https://github.com/FynniX/acc-node-wrapper/blob/6fa0bde696d5f0a190d9a4bbbece821ab8213902/src/index.js#L44

**Comment by FynniX** [comment]
Yes this wrapper isn't up to date. In another version of the wrapper which isn't public I use the sizes:
Physics: 800,
Graphic: 1588,
Static: 756

But in the private version I also use a native addon (There I get this sizes from a struct). But it shouldn't be much to edit this wrapper.

**Comment by FynniX** [comment]
When you would create a pull request then I would add those changes to the repo
